Update CHANGELOG.md (#4384)
JetBrains/compose-multiplatform: CHANGELOG.md

#### Desktop
- [Remove deprecated APIs in `TooltipArea` and `PointerEvent`](https://github.com/JetBrains/compose-multiplatform-core/pull/1029)

#### Web (k/js)

> - Decoys generation was disabled. It means the libraries which depend on earlier Compose Multiplatform version are not compatible with 1.6.0 anymore. Projects which apply  Compose Compiler plugin manually need to remove this argument: `plugin:androidx.compose.compiler.plugins.kotlin:generateDecoys=true` 

Reviewer: ```suggestion
- Libraries which depend on earlier Compose Multiplatform version are not compatible with 1.6.0 anymore. This is because decoys generation was disabled. Projects which apply Compose Compiler plugin manually need to remove this argument: `plugin:androidx.compose.compiler.plugins.kotlin:generateDecoys=true`.
```

We should make the accent on incompatibility.
